app/_src/gateway/kong-enterprise/analytics/index.md

```diff
@@ -31,7 +31,49 @@ When using Vitals, note:
   * If a user does not have access to Vitals data, charts will not display.
 
 ## Vitals API
-Vitals data is available via endpoints on Kong’s Admin API. Access to these endpoints may be controlled via Admin API RBAC. The Vitals API is described in the OAS (Open API Spec, formerly Swagger) file. See a sample here (downloadable file): [`vitals.yaml`](/api/vitals.yaml).
+
+Vitals data is available using Vitals API. Access to these endpoints can be managed using RBAC. 
+
+The Vitals API is described using the Open API Specification. You can view it by downloading the [`vitals.yaml`](/api/vitals.yaml) file. Use the API to retrieve health and traffic information from your cluster.
+
+For example, you can use the API to retrieve the number of requests for a service in a workspace by issuing the following `GET` request: 
+
+```bash
+curl -i -X GET http://localhost:8001/WORKSPACE_NAME/vitals/status_codes/by_service \
+    -d "service_id=030f4ade-69ed-48b0-b3e1-cd088a1f2a09" \
```

Comment:
Are we usually replacing "real" values with some placeholders here? I am not sure.

Reply:
We are moving in that direction to signal to people where they would replace their values, and also to make it clear that the code can't just be copied and ran.